include/GafferSceneUI/StandardLightVisualiser.h

```diff
@@ -74,7 +74,7 @@ class GAFFERSCENEUI_API StandardLightVisualiser : public LightVisualiser
 		static IECoreGL::ConstRenderablePtr pointRays();
 		static IECoreGL::ConstRenderablePtr distantRays();
 		static IECoreGL::ConstRenderablePtr spotlightCone( float innerAngle, float outerAngle, float lensRadius );
-		static IECoreGL::ConstRenderablePtr environmentSphere( const Imath::Color3f &color, const std::string &textureFileName );
+		static IECoreGL::ConstRenderablePtr environmentSphere( const Imath::Color3f &color, const std::string &textureFileName, const IECore::IntData *maxTextureResolution );
```

Comment:
As per the Cortex review, let's use an `int` rather than `IntData`.

Reply:
Addressed in 8c2e916.